hdtv/rootext/fit/ExpBg.cc

```diff
@@ -238,8 +222,7 @@ double ExpBg::_EvalRegion(double *x, double *p) {
   std::list<double>::iterator iter;
 
   bool reject = true;
-  for (iter = fBgRegions.begin(); iter != fBgRegions.end() && *iter < x[0];
-       iter++) {
+  for (iter = fBgRegions.begin(); iter != fBgRegions.end() && *iter < x[0]; iter++) {
```

Comment:
Codacy found an issue: [Prefer prefix ++/-- operators for non-primitive types.](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)